app/views/computacenter/techsource/new.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title, t('page_titles.computacenter.techsource') %>
+<% content_for :before_content, govuk_link_to('Back', computacenter_home_path, class: 'govuk-back-link') %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.computacenter.techsource') %>
+    </h1>
+
+    <p class="govuk-body">Enter email addresses of users that have a TechSource account, there should be one email address per line.</p>
+
+    <%= form_with model: @form, url: computacenter_techsource_path do |f| %>
+      <%= f.govuk_text_area :emails,
+                            label: { text: 'Email addresses' },
+                            hint_text: 'Enter one email address per line',
```

Comment:
Following on from @fofr's comment, I think we can probably do without the hint text - it's just repeating what's in the paragraph above, and potentially in the error message below

Reply:
the hint text has now been removed. some surrounding copy has also been tweaked